src/lib.rs

```diff
@@ -21,6 +21,24 @@ pub enum Interface {
     Any,
 }
 
+pub enum FlowControl {
+    Disable,
```

Comment:
Naming: having this one as the only verb in this list looks weird. I'd prefer something like `Disabled` / `Off`